packages/docs/docs/app/bots.md

```diff
@@ -44,98 +44,81 @@ console.log('Hello world');
+
+The following resources are available to the Bot code, to enable it to do the functionality it requires.
 
 | Name              | Type     | Description                                                                       |
 | ----------------- | -------- | --------------------------------------------------------------------------------- |
-| `resource`        | object   | The FHIR resource that triggered the bot/subscription                             |
-| `console`         | object   | A normal console-like variable that can be used for logging output to AuditEvents |
-| `repo`            | object   | A FHIR repository helper. See below.                                              |
-| `assertOk`        | function | Helper function to validate the outcome of a repository call. See below.          |
-| `createReference` | function | Helper function to create FHIR References from FHIR Resources.                    |
+| `medplum`       | function   | An instance of the medplum JS SDK ([documentation](https://docs.medplum.com/typedoc/core/index.html))                             |
+| `event`         | object   | The event that object that triggered the Bot |
+| `event.input`   | object   | The bot input, usually a FHIR object or content that was posted to a bot endpoint                                              |
+| `console`       | function | Helper function to validate the outcome of a repository call. See below.          |
 
-### resource
-
-The FHIR resource as a plain old JavaScript object.
-
-### console
+### medplum
 
-A `console`-like variable that can be used for logging output to AuditEvents.
+The [Medplum JS client library](https://docs.medplum.com/typedoc/core/index.html), use this library to read and write FHIR resources.  
 
-Example:
+For example, you can use `medplum` in your Bot using the following:
 
 ```javascript
-console.log('Example');
+const patient = await medplum.readResource('Patient', 'fdbaf571-919a-4a08-a671-7dffe4340da8');
 ```
 
-### repo
-
-A FHIR repository helper. The helper exposes a number of utility methods for managing FHIR resources:
-
-#### createResource
-
-`async createResource<T extends Resource>(resource: T): RepositoryResult<T>`
+### event
 
-Creates a resource
+This is a JSON object representing the event that triggered the Lambda.
 
-#### readResource
+### event.input
 
-`async readResource<T extends Resource>(resourceType: string, id: string): RepositoryResult<T>`
+This is the content that was input into the Lambda.  In this example it will be the `Patient` resource because we set up the [Subscription](https://app.medplum.com/Subscription) to fire when the `Patient` resource is edited.
 
-Reads a resource by type and ID
-
-#### updateResource
-
-`async updateResource<T extends Resource>(resource: T): RepositoryResult<T>`
-
-Updates a resource
-
-#### deleteResource
-
-`async deleteResource(resourceType: string, id: string): RepositoryResult<void>`
+### console
```

Comment:
Does `console` even need to be called out a separate thing? Seems like it's just standard JS, no?

Reply:
It's mostly standard.  

Typescript will complain if you don't explicitly add something in `tsconfig.json` to enable it, so it is perhaps worthwhile to clarify that we do support it.

Maybe move it to a separate section for "Logging", and just clarify that we do support `console`, and it shows up in `AuditEvents`.  I think that is good worthwhile info.